portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDeliveryAPI/ExposeErcFromAssetLibraryForComments.testcase

```diff
@@ -44,6 +44,51 @@ definition {
 		}
 	}
 
+	@disable-webdriver = "true"
+	@priority = "4"
+	test CanCreateCommentWithValueInternalIdPreviouslyCreated {
+		property portal.acceptance = "true";
+
+		task ("And Given with POST request I create a comment with a custom external reference code") {
+			var blogPostingId = BlogPostingAPI.getBlogPostingsIdList();
+
+			var response = CommentAPI.createCommentInBlogPosting(
+				blogPostingId = "${blogPostingId}",
+				commentText = "CommentOne",
+				externalReferenceCode = "ercComment");
+		}
+
+		task ("When with POST request I create a comment with a custom external reference code with a value of the internal id of the previously created comment") {
+			var idComment = JSONUtil.getWithJSONPath("${response}", "$.id");
+
+			var response2 = CommentAPI.createCommentInBlogPosting(
+				blogPostingId = "${blogPostingId}",
+				commentText = "CommentTwo",
+				externalReferenceCode = "${idComment}",
+				idComment = "1");
```

Comment:
why do we put an idComment into the createComment endpoint? 🤔

Reply:
Ops. I confused the IDs requested in the test, but I will remove and update the PR. 😅